AngularJSLibrary/__init__.py

```diff
@@ -142,6 +142,14 @@ def __init__(self,
         | Library `|` AngularJSLibrary `|` ignore_implicit_angular_wait=${true}   | # Will not wait for angular syncronization
 
         """
+        global js_wait_for_angular
+        global js_waiting_var
+        global js_get_pending_http_requests
```

Comment:
@lifeeth Why are you using global here? There is other more pythonic ways to do this.

[Sorry for not responding earlier. Somehow I missed this pull request in my inbox.]